README.md

```diff
@@ -147,26 +143,12 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
 I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235645907-ff3ecba3-0626-41cb-ba00-2b321641e5bc.png)

**[OPTIONAL]**

- [ ] It would be nice if you remove this NOTE shown in the image above ⬆️ to make your readme file clean and descriptive 👍